### PR TITLE
Add gcc build dependency

### PR DESCRIPTION
**Pull request by tjorim**
# Proposed Changes

Add gcc build dependency, which seems needed to build the psutil requirement

## Related Issues

Needed for #279

### CODE REVIEW

File: glances/Dockerfile [hunk 1]
```diff
@@ -12,6 +12,7 @@ COPY requirements.txt /tmp/
 # hadolint ignore=DL3042
 RUN \
     apk add --no-cache --virtual .build-dependencies \
+        gcc=11.2.1_git20220219-r2 \
```
Comment (frenck) [inline glances/Dockerfile:15]: This is lower maintenance (changes way less often, as it is a meta package), and also provides gcc.

```suggestion
        build-base=0.5-r3 \
```

**Review by frenck — CHANGES_REQUESTED** [review]

**Review by frenck — APPROVED** [review]
Thanks, @tjorim 👍

**Comment by tjorim** [comment]
Ugh, why don't they have the prebuilt we need anymore. Looks like it also wants `python3-dev` now.

**Comment by frenck** [comment]
> Ugh, why don't they have the prebuilt we need anymore. Looks like it also wants python3-dev now.

Our base images also pick up Home Assistant wheels (https://wheels.home-assistant.io), that is the reason it now needs a bit more (as HA hasn't bumped yet).

**Comment by tjorim** [comment]
Oh I see, I expected the build to check home assistant wheels in addition to pypi. So if a PR is merged in HA core to bump psutil (used for systemmonitor), the wheels for that will also be build.

**Comment by frenck** [comment]
> So if a PR is merged in HA core to bump psutil (used for systemmonitor), the wheels for that will also be build?

Yes!

**Pull request merged**